x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer.ts

```diff
@@ -24,6 +24,11 @@ interface TestData {
   sourceIndexOrSavedSearch: string;
   fieldNameFilters: string[];
   fieldTypeFilters: string[];
+  minimumRowsPerPage?: 10 | 25 | 50;
```

Comment:
I'm not sure about the naming here: why is this prefixed with `minimum`?
This value is used to set the rows per page, so I expected something like `rowsPerPage` (or maybe `maximumRowsPerPage` since it's an upper bound for the displayed rows). But maybe I missed some context here?

Reply:
I think I prefixed it with minimum because that's the pagination we should set so that all fields are displayed in the first page for testing purposes. But seeing how it can be confusing, I updated it to `rowsPerPage` here https://github.com/elastic/kibana/pull/88880/commits/647a5d242c285eec81dc017af8cd3b34f77651a9